popup.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>read-later-notifier</title>
+        <link rel="stylesheet" href="style.css">
+    </head> 
+    <body>
+        <h2>Selected Folder </h2>
+        <blockquote contenteditable="true">
+            
+        </blockquote>
+        <button type="button">
+            Set
+        </button>
+    </body>
+    <body>
+        <h2>Time Period </h2>
```

Comment:
The initial character of "Period" and "Folder" should be lowercase to collect in the same series.